# **Hypothesis Testing**

Background:

Bombay hospitality Ltd. operates a franchise model for producing exotic Norwegian dinners throughout New England. The operating cost for a franchise in a week (W) is given by the equation W = $1,000 + $5X, where X represents the number of units produced in a week. Recent feedback from restaurant owners suggests that this cost model may no longer be accurate, as their observed weekly operating costs are higher.

Objective:

To investigate the restaurant owners' claim about the increase in weekly operating costs using hypothesis testing.

Data Provided:

The theoretical weekly operating cost model: W = $1,000 + $5X

Sample of 25 restaurants with a mean weekly cost of Rs. 3,050

Number of units produced in a week (X) follows a normal distribution with a mean (μ) of 600 units and a standard deviation (σ) of 25 units

**Assignment Tasks:**

1. State the Hypotheses statement
2. Calculate the Test Statistic
3. Determine the Critical Value
4. Make a Decision
5. Conclusion

**Step 1: State the Hypotheses Statement**

Null Hypothesis (
𝐻
0
H
0
​
 ): The mean weekly operating cost follows the theoretical model, i.e.,
𝜇
𝑊
=
1
,
000
+
5
𝜇
𝑋
=
4
,
000
μ
W
​
 =1,000+5μ
X
​
 =4,000.

Alternative Hypothesis (
𝐻
𝑎
H
a
​
 ): The mean weekly operating cost is higher than the theoretical model, i.e.,
μ W>4,000.

**Step 2: Calculate the Test Statistic**

In [ ]:
import numpy as np
from scipy.stats import t

# Given data
sample_mean = 3050  # Sample mean weekly cost
theoretical_mean = 4000  # Theoretical mean weekly cost
n = 25  # Sample size
sigma_X = 25  # Population standard deviation of units produced

# Calculate the standard error
sigma_W = 5 * sigma_X
standard_error = sigma_W / np.sqrt(n)

# Calculate the t statistic
t_statistic = (sample_mean - theoretical_mean) / standard_error
t_statistic


-38.0

**Step 3: Determine the Critical Value**

The critical value for a one-tailed t-test at a 0.05 significance level with 24 degrees of freedom (n - 1 = 25 - 1 = 24) can be found using the t-distribution.

In [ ]:
# Significance level
alpha = 0.05

# Degrees of freedom
df = n - 1

# Critical value
critical_value = t.ppf(1 - alpha, df)
critical_value


1.7108820799094275

**Step 4: Make a Decision**

Compare the t statistic with the critical value to make a decision.

In [ ]:
# Decision
if t_statistic > critical_value:
    decision = "Reject the null hypothesis"
else:
    decision = "Fail to reject the null hypothesis"

decision


'Fail to reject the null hypothesis'

In [ ]:
# Print results
print(f"Chi-Square Statistic: {t_statistic}")
print(f"Critical Value: {critical_value}")
print(f"Decision: {decision}")

Chi-Square Statistic: -38.0
Critical Value: 1.7108820799094275
Decision: Fail to reject the null hypothesis


**Step 5: Conclusion**

Based on the calculated t statistic (-38.0) and the critical value (1.7109), we fail to reject the null hypothesis at the 0.05 significance level. This means that we do not have enough evidence to support the claim that the mean weekly operating cost is higher than the theoretical model suggests. The observed data does not indicate a significant increase in weekly operating costs based on the sample provided.